packages/xstate-wallet/src/utils/contract-utils.ts

```diff
@@ -23,6 +23,9 @@ export function tokenAddress(assetHolderAddress: string): string | undefined {
 
 export function getProvider(): Web3Provider | JsonRpcProvider {
   if (window.ethereum) {
+    if (window.ethereum.mockingInfuraProvider) {
```

Comment:
I wish we didn't need to reference mocks in production code 🤷

Reply:
Yeah I definitely agree. I started on some work to rip out the provider as a configuration option but haven't finished it yet.